lib/racc/directed_graph.rb

```diff
@@ -351,11 +348,11 @@ def reachable
       end
 
       def reachable_from(nodes)
-        Racc.set_closure(nodes) { |node| node.out }
+        Racc.set_closure(nodes) { |node, _| node.out }
       end
 
       def can_reach(dests)
-        Racc.set_closure(dests) { |node| node.in }
+        Racc.set_closure(dests) { |node, _| node.in }
```

Comment:
This code cannot autofixed. so I added some fix.

set_closure take 2 arguments for block. In that case, it seems not to write following.

```
Racc.set_closure(dests, &:in)
```